docs/content/docs/test/Runner.md

```diff
@@ -15,7 +15,43 @@ It can do the following:
 - Save tests result in a report.\
 It uses an internal TMS (test management system). We are working on support other formats.
 
-## Example
+## Getting started
+
+TBD
+
+## Configuration
+
+A configuration is done in gradle script:
+
+```kotlin
+plugins {
+    id("com.android.application")
+    id("com.avito.android.instrumentation-tests")
+}
+
+extensions.getByType<GradleInstrumentationPluginConfiguration>().apply {
+    // all options goes here
+}
+
+```
+
+### Test APK
+
+Plugin builds APKs on his own by default.\
+If for any reason you have to build APK externally, you can pass files manually:
```

Comment:
I see probable reasons to pass APK:
- It has been built externally on another host due to performance
- It has been transformed due to security reasons and we can't get a final artifact from AGP

Reply:
i think this comment could be a part of documentation

Reply:
It is a documentation